django/book/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from .models import Book
+
+
+class BookSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Book
+        fields = '__all__'
```

Comment:
nit: better to put all field names explicitly to see what actually is going to be used